src/lsp/cobol_parser/parser_engine.mli

```diff
@@ -119,12 +119,10 @@ val rewind_and_parse
 
 (** {1 Rewinding for inspection} *)
 
-module INSPECT: module type of Grammar.MenhirInterpreter
```

Comment:
Please leave that one as is (it's internal and makes the type definition below a bit clearer).